cups/http-addr.c

```diff
@@ -207,7 +207,7 @@ httpAddrListen(http_addr_t *addr,	// I - Address to bind to
   // Create the socket and set options...
   if ((fd = socket(addr->addr.sa_family, SOCK_STREAM, 0)) < 0)
   {
-    _cupsSetHTTPError(HTTP_STATUS_ERROR);
+    _cupsSetHTTPError(NULL, HTTP_STATUS_ERROR);
```

Comment:
Use `_cupsSetError(IPP_STATUS_ERROR_INTERNAL, strerror(errno), 0);` for these cases, that way `_cupsSetHTTPError` can be specific to `http_t` errors.

Reply:
ok, thanks! Applied.